gh_test.go

```diff
@@ -85,25 +86,26 @@ func TestGQLClient(t *testing.T) {
 	os.Setenv("GH_CONFIG_DIR", tempDir)
 	os.Setenv("GH_TOKEN", "GH_TOKEN")
 
-	http := httpmock.NewRegistry(t)
-	http.Register(
-		httpmock.GQL("QUERY"),
-		httpmock.StringResponse(`{"data":{"viewer":{"login":"hubot"}}}`),
-	)
+	gock.New("https://api.github.com").
+		Post("/graphql").
+		MatchHeader("Authorization", "token GH_TOKEN").
+		BodyString("QUERY").
```

Comment:
Perhaps we could define a custom matcher that matches the query name in a GraphQL request by parsing the JSON and looking solely at the `query` value, rather than at the whole body of the request

Reply:
We could do that. Do you feel that approach provides significant additional value? It feels like it comes down to the same matching mechanism in the end.